ci-operator/step-registry/cucushift/installer/check/ibmcloud/cucushift-installer-check-ibmcloud-commands.sh

```diff
@@ -29,6 +64,17 @@ if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
-checkProviderType
+check_result=0
+
+if ! checkProviderType ; then
+    check_result=1
+fi
+
+if [ -s "${SHARED_DIR}/ibmcloud_cluster_resource_group" ]; then
+    ibmcloud_login
+    if ! checkResourcesInVPC ; then
```

Comment:
Similar https://github.com/openshift/release/pull/51310/files#r1582469806, and the current step is `cucushift-installer-check-ibmcloud` step, it is supposed to be used in common cases, but this check sounds like be specific for OCP-60816.

Reply:
How about adding checking method for some cases in cucushift-installer-check-ibmcloud-commands.sh?

Reply:
I do not suggest to do that, multiple steps can help us easily assemble a new install workflow per our demand.